src/styles.css

```diff
@@ -516,7 +517,7 @@ filter: drop-shadow(5px 10px 3px black);
        justify-content: center;
        align-items: center;
        height: 10%;
-       width: 25%; 
+       width: 25%;
        margin: 0;
```

Comment:
Need left margin to create space between images. Possibly adjust in the images section as well.